### PR TITLE
homePage

### PR DESCRIPTION
**Pull request by preethasri**
homepage of ecommerce site where you can see all the categories listed to buy

### CODE REVIEW

**Review by BelideNikhil — APPROVED** [review]
Try using REM for margin and padding.
Also save colors and font size as variables as well.
Rest is good.

**Review by poornima-alugubelly — APPROVED** [review]
I think u can put break point for search bar before it gets squished like this 
image

File: homepage/homepage.css [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+/* image */
+
+.landing-page-image {
+    position: relative;
+}
+
+.main-img {
+    width: 100%;
+    height: 50vh;
+}
```
Comment (MiheerTamkhane) [inline homepage/homepage.css]: On main img use object-fit : cover, it will set respective parent and will look better.

**Review by MiheerTamkhane — APPROVED** [review]
Looks good to me

**Review by Chakravarthi0 — APPROVED** [review]
You could use rem for values greater than 10px and variables for colors.

Rest looks good to me.

File: homepage/homepage.css (hunk 1 above)
Comment (preethasri) [reply homepage/homepage.css]: > On main img use object-fit : cover, it will set respective parent and will look better.

thanks miheer will look on it

**Comment by preethasri** [comment]
> You could use rem for values greater than 10px and variables for colors.
> 
> Rest looks good to me.

will work on that saran

**Comment by preethasri** [comment]
> I think u can put break point for search bar before it gets squished like this image

thanks poornima. i will look at it and put breakpoints for tab too.thank you

**Pull request merged**